scipy/conftest.py

```diff
@@ -159,9 +159,17 @@ def check_fpu_mode(request):
                 msg = f"'--array-api-backend' must be in {xp_available_backends.keys()}"
                 raise ValueError(msg)
 
+
 if 'cupy' in xp_available_backends:
     SCIPY_DEVICE = 'cuda'
 
+    # this is annoying in CuPy 13.x
+    warnings.filterwarnings(
+        'ignore', 'cupyx.jit.rawkernel is experimental', category=FutureWarning
+    )
+    from cupyx.scipy import signal
+    del signal
```

Comment:
is this still needed?

Reply:
Yes. it emits the warning on import of cupyx.signal.